src/_product-updates/2024-07-25-access-management.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Access management
+tags: [Improvement]
+image: /images/changelog/changelog-access-management.png
+---
+
+![changelog-entry-page.png](/images/changelog/changelog-access-management.png)
+
+It's been a while since we added the first pillar of our access management system. It gave admins a way to precisely manage their users' access.
```

Comment:
```suggestion
It's been a while since we added the first pillar of our access management system. It gave documentation maintainers a way to grant access to internal users ("members") and external users ("guests").
```